django/db/models/sql/query.py

```diff
@@ -1112,16 +1112,16 @@ def check_related_objects(self, field, value, opts):
                 for v in value:
                     self.check_query_object_type(v, opts, field)
 
-    def check_filterable(self, expression):
+    def _check_filterable(self, expression):
         """Raise an error if expression cannot be used in a WHERE clause."""
-        if not getattr(expression, 'filterable', 'True'):
+        if not getattr(expression, 'filterable', True):
```

Comment:
Good catch, this cleanup should be moved to a separate commit.

Reply:
Done